components/Reviews/components/AllReviewsLink/index.jsx

```diff
@@ -7,30 +7,38 @@
 
 import React from 'react';
 import PropTypes from 'prop-types';
+import { bin2hex } from '@shopgate/pwa-common/helpers/data';
 import I18n from '@shopgate/pwa-common/components/I18n';
 import Link from '@shopgate/pwa-common/components/Router/components/Link';
 import { REVIEW_PREVIEW_LIMIT } from 'Pages/Product/constants';
 import connect from './connector';
-import styles from './style';
+import {
+  containerStyle,
+  linkStyle,
+} from './style';
 
 /**
  * Link to all reviews.
+ * @param {string} productId ProductId.
  * @param {number} totalReviewCount The number of reviews given.
```

Comment:
Param description is incorrect. Is this not the total number of reviews that a product has?

Reply:
True. Adjusted.